Update dashboard document for Kubernetes environment (#4712)
wso2/docs-apim: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md

    group_id = "mi_dev"
    node_id = "dev_node_2"
    ```
    

>     If MI server is deployed in a Kubernetes environment, add following configuration too to the `deployment.toml`. 

Reviewer: ```suggestion
    If the Micro Integrator server is deployed in a Kubernetes environment, add the following configuration to the `deployment.toml` file. 
```

    
    Note: This capability is released as a product update on todo. If you do not have this update, you can [get the latest updates](https://updates.docs.wso2.com/en/latest/updates/overview/#!) now.
    

>     Limitation: Currently the dashboard does not support deployments with replicas

Reviewer: ```suggestion
    Limitation: Currently the dashboard does not support deployments with replicas.
```

>     If communication via Ingress:

Reviewer: ```suggestion
    If communicating via Ingress, use the following configuration:
```

    ```toml
    dashboard_url = "https://{hostname/ip}:{port}/dashboard/api/"
    management_hostname = "<INGRESS_HOSTNAME>"
    ```
    

>     If communication via Service

Reviewer: ```suggestion
    If communicating via Service instead, use the following configuration:
```
